### PR TITLE
Configures release train

### PR DESCRIPTION
**Pull request by vegaro**
- Added release trains
- Added manual triggers
- Added workflow to be triggered remotely when updating purchases-hybrid-common

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -145,13 +166,16 @@ jobs:
...
+      xcode: 13.4.0
     steps:
       - checkout
       - run:
           name: Start simulator
           command: xcrun instruments -w "iPhone 13 (15.4) [" || true
       - npm-dependencies
+      - run:
+          name: Install Cordova Paramedic
+          command: npm install --location=global github:apache/cordova-paramedic#c394568c093b13da6a1fe69546aef0a66c21a7de
```
Comment (tonidero) [inline .circleci/config.yml:178]: Could this be moved to a `command`? To avoid repetition

File: fastlane/Fastfile [hunk 2]
```diff
@@ -77,26 +105,32 @@ lane :update_hybrid_common_versions do |options|
...
+    current_version: current_phc_version,
+    next_version: new_version_number,
+    open_pr: options[:open_pr] || false,
+    automatic_release: options[:automatic_release] || false
+  )
+end
+
+desc "Update Hybrid Common and builds examples. Calls update_hybrid_common, then build_hybrid_example and pushes changes to a new branch if open_pr option is true"
+lane :update_hybrid_common do |options|
+  update_hybrid_common_versions(options)
```
Comment (tonidero) [inline fastlane/Fastfile:133]: Should we just rename the `update_hybrid_common_versions` to `update_hybrid_common`?

**Review by tonidero — APPROVED** [review]
Looks good! Just a couple of small things.

File: fastlane/Fastfile (hunk 2 above)
Comment (vegaro) [reply fastlane/Fastfile:133]: oh yes, I did that in react native, but not here

**Pull request merged**